src/main/java/use_case/add_assignment/AddAssignmentInteractor.java

```diff
@@ -8,7 +8,7 @@
  * The Add Assignment Interactor.
  */
 public class AddAssignmentInteractor implements AddAssignmentInputBoundary {
-    private final AddAssignmentDataAccessInterface assignmentDataAccessObject;
+    private final DataAccessInterface assignmentDataAccessObject;
```

Comment:
FYI that since the database user DAO is now created we're going to transition to using that vs. the DataAccessInterface in memory one (make sure to pull from main to get that DAO!)